doc/unbound.conf.5.in

```diff
@@ -738,7 +738,7 @@ the cache contents (for malicious acts).  However, nonrecursive queries can
 also be a valuable debugging tool (when you want to examine the cache
 contents). In that case use \fIallow_snoop\fR for your administration host.
 .IP
-The \fIallow_cookie\fR action allows access to UDP queries that contain a
+The \fIallow_cookie\fR action allows only access to UDP queries that contain a
```

Comment:
I would make it "allows access only" rather than "allows only access".

Reply:
Done in commit 16fe26f